x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/elastic_endpoint_security_process_injection_detected.json

```diff
@@ -6,7 +6,6 @@
   ],
   "interval": "10m",
   "language": "kuery",
-  "max_signals": 100,
```

Comment:
Is this supposed to disappear?

Reply:
we actually need to put that back b/c we will set it in future rules when we have more complex thresholding and it would be good to track what we set. but it will be set to the default of 100.